openfilter/lineage/schema/OpenFilterConfigRunFacet.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/PlainsightAI/openfilter/schema/lineage/OpenFilterConfigRunFacet.json",
```

Comment:
You may also want to follow the [versioning semantics](https://github.com/OpenLineage/OpenLineage/blob/main/spec/Versioning.md) for OpenLineage `MODEL-REVISION-ADDITION`, but that's only if you wanted to publish the schema under `https://openfilter.io/openlineage/facets/MODEL-REVISION-ADDITION/OpenFilterConfigRunFacet.json`. Maybe we leave as if for now.